org.eclipse.xtend.ide/src/org/eclipse/xtend/ide/buildpath/XtendClasspathContainer.java

```diff
@@ -87,7 +87,16 @@ private IPath bundlePath(Bundle bundle) {
 	}
 
 	private IPath binFolderPath(Bundle bundle) {
-		URL binFolderURL = FileLocator.find(bundle, new Path("bin/main"), null); //$NON-NLS-1$
+		URL binFolderURL = FileLocator.find(bundle, new Path("target/classes"), null); //$NON-NLS-1$
+		if (binFolderURL != null) {
+			try {
+				URL binFolderFileURL = FileLocator.toFileURL(binFolderURL);
+				return new Path(binFolderFileURL.getPath()).makeAbsolute();
+			} catch (IOException e) {
+				LOG.error("Can't resolve path '" + bundle.getSymbolicName() + "'", e); //$NON-NLS-1$ //$NON-NLS-2$
+			}
+		}
+		binFolderURL = FileLocator.find(bundle, new Path("bin/main"), null); //$NON-NLS-1$
```

Comment:
should we remove the two bin options?

Reply:
bin/main is history so I'd keep only the new one